lib/linguist/languages.yml

```diff
@@ -1244,6 +1244,16 @@ Emacs Lisp:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 102
+Email:
+  type: data
+  extensions:
+  - ".eml"
+  - ".email"
+  - ".mail"
+  - ".mime"
```

Comment:
I'm not sure I understand what this extension has to do with this...?

Reply:
@pchaigno Those were the extensions in the textmate grammar. I'll remove the last three